syft/pkg/license.go

```diff
@@ -25,9 +25,15 @@ var _ sort.Interface = (*Licenses)(nil)
 // in order to distinguish if packages should be kept separate
 // this is different for licenses since we're only looking for evidence
 // of where a license was declared/concluded for a given package
+// If a license is given as it's full text in the metadata rather than it's value or SPDX expression
+
+// The FullText field is used to represent this data
+// A Concluded License type is the license the SBOM creator believes governs the package (human crafted or altered SBOM)
+// The Declared License is what the authors of a project believe govern the package. This is the default type syft declares.
 type License struct {
-	Value          string
 	SPDXExpression string
+	Value          string
+	FullText       string
```

Comment:
nit on naming: I don't think we should claim that the value is the "full text", but instead maybe just the "contents"

Reply:
These are good suggestions for 2.0 when we revisit the model again